version/1.0.0/docs/1.0.0.html

```diff
@@ -37,7 +37,7 @@ <h1 class="logo">MarkdownTag<span class="logo__thin">Docs</span></h1>
       <aside class="doc__nav">
         <ul>
           <li class="js-btn selected" id="nav_btn-get_started">Get Started</li>
-		  <li class="js-btn" id="nav_btn-config">Instalation</li>
+		  <li class="js-btn" id="nav_btn-config">Installation</li>
```

Comment:
Same thing here - either spelling might be fine, not sure if you want to keep the previous spelling @DarkenLM

Reply:
That one was a typo. To keep consistency, I think it should be typed as "Installation".

Reply:
> The page at github.com/DarkenLM/Markdown-Tag/index.html renders fine on my Samsung Galaxy S22+

@shaunbharat was referring to the documentation demo @DarkenLM made with the preview selector

Reply:
That appears to be fine on my S22+ too

Reply:
@shaunbharat - thanks for confirming. 

Sorry to ask again but just to clarify to - you are looking at the index page I mean tho? With the demo of Markdown-Tag in use? Not the docs page.

Reply:
Yes, I checked that and it seems to render correctly on my S22+